src/actions/commands/actions.ts

```diff
@@ -371,7 +371,7 @@ class CommandInsertRegisterContentInSearchMode extends BaseCommand {
 @RegisterAction
 class CommandRecordMacro extends BaseCommand {
   modes = [ModeName.Normal, ModeName.Visual, ModeName.VisualLine];
-  keys = ['q', '<character>'];
+  keys = [['q', '<alpha>'], ['q', '<number>']];
```

Comment:
I supposed we should also allow `q"` here.

Reply:
Thanks for pointing out bugs.
I fix this problem(0a635c35a225c39ed1dc2a8b6d1e5e6010183368).